app/pages/sidedrawer.xml

```diff
@@ -14,8 +16,38 @@
       </ActionBar.actionItems>
     </ActionBar>
   </Page.actionBar>
-  <StackLayout class="demo-full">
-    <Label text="TO DO :)" textWrap="true" />
-     
-  </StackLayout>
+
+  <drawer:RadSideDrawer id="sideDrawer" drawerLocation="Left">
+      <drawer:RadSideDrawer.mainContent android:showOverNavigation="false">
+      <StackLayout class="mainContent">
+            <Label text="Side Drawer Example" />
+      </StackLayout>
+    </drawer:RadSideDrawer.mainContent>
+    <drawer:RadSideDrawer.drawerContent >
```

Comment:
Might need to wrap in `ScrollView` for lengthy lists and certain devices with smaller screen heights.